src/qt/walletmodel.cpp

```diff
@@ -120,12 +123,18 @@ void WalletModel::pollBalanceChanged()
 
 void WalletModel::checkBalanceChanged(const interfaces::WalletBalances& new_balances)
 {
-    if(new_balances.balanceChanged(m_cached_balances)) {
+    const auto& cached_balance = getCachedBalance();
+    if (new_balances.balanceChanged(cached_balance)) {
         m_cached_balances = new_balances;
         Q_EMIT balanceChanged(new_balances);
     }
 }
 
+interfaces::WalletBalances WalletModel::getCachedBalance() const
```

Comment:
08e74071e3ae30393a80d94a8286536ad7bfab3d

Without having a setter, maybe shorter `cachedBalance()` is better?

Reply:
At least for me, it's cleared to always use the same naming convention even if we don't have a setter right now.

I mean, generally speaking, if the method is a getter, then why not just name it `getSomething`?
So any dev using the model interface (or any interface) can find it right away without even need to think twice about it.

At least for me, those three characters make searches across large interfaces/projects simpler.